applications/dashboard/src/scripts/compatibilityStyles/buttonStylesCompat.ts

```diff
@@ -41,42 +41,64 @@ export const buttonCSS = () => {
     mixinButton(".BigButton:not(.Danger)", ButtonTypes.PRIMARY);
     mixinButton(".NewConversation.NewConversation", ButtonTypes.PRIMARY);
     mixinButton(".groupToolbar .Button.Primary", ButtonTypes.PRIMARY);
-    mixinButton(".BoxButtons .Button.Primary", ButtonTypes.PRIMARY);
+    mixinButton(".BoxButtons .ButtonGroup.Multi .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".Section-Members .Group-RemoveMember", ButtonTypes.PRIMARY);
     mixinButton(".group-members-filter-box .Button.search", ButtonTypes.PRIMARY);
     mixinButton("#Form_Ban", ButtonTypes.PRIMARY);
     mixinButton(".Popup #UserBadgeForm button", ButtonTypes.PRIMARY);
     mixinButton(".Button.Handle", ButtonTypes.PRIMARY);
     mixinButton("div.Popup .Body .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".ButtonGroup.Multi .Button.Handle", ButtonTypes.PRIMARY);
-    mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
+    // mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
+
+    const buttonBorderRadius = parseInt(formElementVars.border.radius.toString(), 10);
```

Comment:
Shouldn't this be the `buttonGlobalVars().border.radius`?

Reply:
You're right, it's not the right value. however, it should be globalVars.borderType.formElements.buttons.

This is the hierarchy:
```
    const border = makeThemeVars("border", {
        color: mixBgAndFg(0.15),
        colorHover: mixBgAndFg(0.2),
        width: 1,
        style: "solid",
        radius: 6, // Global default <=== Default for all
    });

    const borderType = makeThemeVars("borderType", {
        formElements: {
            default: border, // <=== Default for form elements (like inputs)
            buttons: border, // <=== Defaults for buttons
        },
        modals: border, // <=== Defaults for modals
        dropDowns: {
            content: border, // <=== Defaults for dropdown content
        },
    });
```